comp/dogstatsd/server/intern.go

```diff
@@ -66,30 +69,28 @@ func (i *stringInterner) LoadOrStore(key []byte) string {
 	// See https://github.com/golang/go/commit/f5f5a8b6209f84961687d993b93ea0d397f5d5bf
 	if s, found := i.strings[string(key)]; found {
 		if i.tlmEnabled {
-			tlmSIRHits.Inc()
+			tlmSIRHits.WithTags(map[string]string{"internerId": i.id}).Inc()
```

Comment:
Not that it matters much, but we should probably use snake case for the tag, `interner_id`, it'll be more consistant with the rest in graphs.